src/rpm/builder.rs

```diff
@@ -151,7 +152,9 @@ impl RPMBuilder {
                 .modified()?
                 .duration_since(UNIX_EPOCH)
                 .expect("something really wrong with your time")
-                .as_secs() as i32,
+                .as_secs()
+                .try_into()
+                .unwrap(),
```

Comment:
This goes for every instance - should this be handled (by conversion into an RPMError) or expect()ed ? Or are we assuming that this is completely infallible (what happens if we parse a file with a modified date in 2040 for example - or is this just a limitation of the RPM spec?)

Reply:
Well realistically for that purpose RPM needs to stop storing the tag as a 32 bit integer.  If all the integers are unsigned then that means it should be OK for any date from 1970 out to about 2100 so there is at least no "year 2038 problem"

https://github.com/rpm-software-management/rpm/issues/1084

Reply:
Generally speaking, an `RPMError`. In this particular case, I think a `expect()` should suffice.